docs/eks-v3-migration.md

```diff
@@ -142,22 +142,57 @@ A step-by-step migration guide can be found [here](https://github.com/pulumi/pul
-  - Creating an IRSA based IAM role now requires you to use `apply` for accessing the OIDC provider ARN and URL. An example of how this works can be found [here](https://github.com/pulumi/pulumi-eks/blob/release-3.x.x/examples/oidc-iam-sa/index.ts).  
-  - `k8s.Provider` will be deleted if not referenced (no impact, but it will appear in the diff)
-  - `clusterOidcProvider` is an output now. `getKubeConfig` returns an output now  
-- The deprecated input property `deployDashboard` of the `Cluster` component has been removed from the Nodejs SDK. This has already been removed from the other SDKs in the past. If you’d like to continue using it, you can adopt the existing code into your own program from [here](https://github.com/pulumi/pulumi-eks/blob/bcc170e72b802a78e7f0a99bc92316a5f8a62b0e/nodejs/eks/dashboard.ts).
+The Node.js SDK is updated to use state of the art Pulumi tooling, improving stability, documentation and security. The update requires the following changes to programs:
+- Properties of the components are now outputs instead of plain types. Accessing those optional resource outputs now requires using `apply`. More details about this in the next section. 
+- The [`Cluster.getKubeConfig`](https://www.pulumi.com/registry/packages/eks/api-docs/cluster/#method_GetKubeconfig) method now returns an output. 
+- The `cluster.provider` will be deleted if not referenced (no impact, but it will appear in the diff)
+- The deprecated input property `deployDashboard` of the `Cluster` component has been removed from the Node.js SDK. This has already been removed from the other SDKs in the past. If you’d like to continue using it, you can adopt the existing code into your own program from [here](https://github.com/pulumi/pulumi-eks/blob/bcc170e72b802a78e7f0a99bc92316a5f8a62b0e/nodejs/eks/dashboard.ts).
 - The `createManagedNodeGroup` function will now create an Pulumi EKS `ManagedNodeGroup` instead of creating the underlying `aws.eks.NodeGroup` resource directly. During the upgrade to Pulumi EKS v3 you'll see the additional wrapper component being created.
 - The capitalization of the `AuthenticationMode` and `AccessEntryType` enum values has been aligned with other providers (e.g. `AuthenticationMode.API` => `AuthenticationMode.Api`). The previous values have been marked as deprecated and now point to their new counterparts.
 
+### Properties of the components are now outputs instead of plain types
+All properties of components in the Node.js SDK are now wrapped in `Output`. This aligns it with other language SDKs but introduces some breaking changes for optional resource properties.
+Accessing those optional resource outputs now requires using `apply`.
+
+To work around this we've exposed the most commonly used resource properties at the top level of the components. This change allows for easier access without requiring to use `apply`.
```

Comment:
There's great information in this paragraph but it is a little verbose, try to do a pass and remove words that can be removed, otherwise it is reasonable.

Perhaps instead of "To work around this", "We added convenient access properties for frequently used outputs:".